setup.py

```diff
@@ -31,7 +31,6 @@
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: Implementation',
         'Programming Language :: Python :: Implementation :: CPython',
-        'Programming Language :: Python :: Implementation :: Jython',
```

Comment:
This change is wrong. A similar one has been rejected some time ago.

Reply:
Please explain.  This repo does not support Python 2 and Jython does not support Python 3.

Reply:
Jason's opinion is that if Jython 3 ever becomes stable, our code will support it. Anyway, this has nothing to do with Circle CI and so mustn't be in the same PR.

Reply:
LOL

Reply:
Well, this is totally reasonable: CherryPy is pure-python and this is why it's expected for it to work under any Python runtime.